### ISSUE TITLE
allow user to custom Stealth Mode for each website 

### ISSUE THREAD

**Issue by M-Makise-Kurisu** [AdguardTeam/AdguardForMac#1030]
Sometime some functions in Stealth Mode may broke some website. 

For example: if I turn on Hide Referrer from third-parties feature in Miscellaneous. https://www.pixivision.net/ will not work properly. 
 
in current version I have two ways to make the website work, turn off Hide Referrer from third-parties feature or turn off adGuard for https://www.pixivision.net/ 

But both way will coast some problem. if I turn off Hide Referrer from third-parties. I may face weaker protection on other website. 
if I turn off adGuard for the website. I may not able to joy the benefit from adGuard on that website.

For the solution I think we can allow user to custom Stealth Mode for each website in browser's adGuard extension, just like how we  select additional element.
or, we can make a list beside each protection in adGuard setting page. if the protection is off, adGuard will automatically turn on when user are on the website in the list. and do the opposite if the protection is on.

**Comment by Chinaski1** [comment]
Hello!

You can use `$stealth` rule to exclude [stealth mode](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#stealth) for certain sites.

 In your case, you need to add  `@@||pixivision.net^$stealth`

**Comment by M-Makise-Kurisu** [comment]
Hi
Thank you for replying.
I got some problem with the rule. 
I tried to add the rule you provide to me, but seems like it does not working.
after that I looked up our website, and got the rule below:
@@||www.pixivision.net^$stealth,removeparam,cookie
but this does not work either.
I tried to change the domain, seems like it's not certain website's problem. 
Do I suppose to write the rule in User rules filter?
what I did wrong?
